### PR TITLE
Fix: case classes must have a non-implicit parameter list

### PR DESCRIPTION
**Pull request by regadas**
This triggers mima with the same issue we have seen when updating mima to the last version.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -197,6 +197,8 @@ lazy val mimaSettings = Def.settings(
       .exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.HeavyHittersPercent.apply"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.BufferSize.apply"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.Compact.apply"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.MinAggregator.ord"),
```
Comment (nevillelyh) [inline build.sbt]: I guess it's backwards compat as long as people don't call `aggr.ord`?

**Review by nevillelyh — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (regadas) [reply build.sbt]: yeah right, with the previous syntax that was totally possible;

Comment (johnynek) [reply build.sbt]: I’m nervous about each of these. I think we should be very reluctant to risk binary incompatibility to clean up warts.

Comment (nevillelyh) [reply build.sbt]: @johnynek yeah those change were reverted in 2nd commit of the PR. The squashed commit doesn't have any MiMa changes.

Comment (johnynek) [reply build.sbt]: Ahh. Right. Good point.  Thanks.

Comment (regadas) [reply build.sbt]: Yup ended up achieving the same thing; @johnynek was the access to `ord` intended by design?

Comment (johnynek) [reply build.sbt]: I see no reason to hide it, you can reconstruct it from max or min:

```
def lteq(a: A, b: A) = max(a, b) == b
```

**Pull request merged**